dev-client/src/components/FeatureFlagControl.tsx

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright © 2024 Technology Matters
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published
+ * by the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program. If not, see https://www.gnu.org/licenses/.
+ */
+import {useState} from 'react';
+import {StyleSheet, View} from 'react-native';
+import {Divider, Switch} from 'react-native-paper';
+
+import {
+  Heading,
+  Text,
+} from 'terraso-mobile-client/components/NativeBaseAdapters';
+import {APP_CONFIG, MMKV} from 'terraso-mobile-client/config';
+
+export const FeatureFlagControl = () => {
+  // It may cause issues to toggle a feature flag state while the app is running,
+  // so expect the user to restart the app if they want to change the feature flag state.
+  // This should never be exposed in production.
+
+  const currentFlagState = APP_CONFIG.FF_offline;
```

Comment:
question: is there a standard naming convention for feature flags? Should it be in all-caps? Should it have a more descriptive name, like FF_OFFLINE_SUPPORT_ENABLED?

Reply:
Resolving because this comment is duplicated above :)